package.json

```diff
@@ -6,13 +6,13 @@
     "build:client": "cd client && cross-env INLINE_RUNTIME_CHUNK=false react-scripts build",
     "build:ssr": "cd ssr && webpack --config webpack.config.js --mode=production",
     "dev": "yarn prepare-build && nf -j Procfile.dev start",
-    "eslint": "eslint package.json 'content/**/*.js' 'build/**/*.js' 'kumascript/src/**/*.js' 'tool/**/*.js' 'server/**/*.js'",
+    "eslint": "eslint .",
     "filecheck": "cd filecheck && node cli.js",
     "fiori:build": "cd client && build-storybook",
     "fiori:start": "cd client && start-storybook -p 6006",
     "prepare-build": "yarn build:client && yarn build:ssr && yarn build --spas-only && yarn tool gather-git-history --verbose",
-    "prettier-check": "prettier --check '**/*.{js,ts,tsx,json,scss,html}'",
-    "prettier-format": "prettier --write '**/*.{js,ts,tsx,json,scss,html}'",
+    "prettier-check": "prettier --check \"**/*.{js,ts,tsx,json,scss,html}\"",
```

Comment:
This is for proper glob expansion on Windows

Reply:
So prior to your fix here, running `yarn prettier-check` wouldn't work on Windows??